docs/AboutUs.md

```diff
@@ -1,51 +1,31 @@
 # About Us
 
-We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
+We are a group of students in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
 
 ## Project Team
 
-#### [Damith C. Rajapakse](http://www.comp.nus.edu.sg/~damithch) <br>
-<img src="images/DamithRajapakse.jpg" width="150"><br>
-**Role**: Project Advisor
+#### [Ang Shi Ya](https://github.com/AngShiYa) <br>
+<img src="images/ShiYa.png" width="150"><br>
+Role: Testing, Developer <br>  
+Responsibilities: Model
 
 -----
 
-#### [Joshua Lee](http://github.com/lejolly)
-<img src="images/JoshuaLee.jpg" width="150"><br>
-Role: Developer <br>  
-Responsibilities: UI
+#### [Wu Cheuk Kwan Carina](https://github.com/carinackwu) <br>
+<img src="images/Carina.JPG" width="150"><br>
+Role: Team Lead, Developer <br>  
+Responsibilities: Logic
 
 -----
 
-#### [Leow Yijin](http://github.com/yijinl) 
-<img src="images/LeowYijin.jpg" width="150"><br>
-Role: Developer <br>  
-Responsibilities: Data
+#### [Huang Tsai Ti](https://github.com/tyrahuang) <br>
+<img src="images/TsaiTi.jpg" width="150"><br>
+Role: Code quality, Developer <br>  
+Responsibilities: Storage
 
 -----
 
-#### [Martin Choo](http://github.com/m133225)
-Role: Developer <br>  
-Responsibilities: Dev Ops
-
------
-
-#### [Thien Nguyen](https://github.com/ndt93)
- Role: Developer <br>  
- Responsibilities: Threading
- 
- -----
-
-#### [You Liang](http://github.com/yl-coder) 
-<img src="images/YouLiang.jpg" width="150"><br>
- Role: Developer <br>  
- Responsibilities: UI
- 
- -----
-
-# Contributors
-
-We welcome contributions. See [Contact Us](ContactUs.md) page for more info.
-
-* [Akshay Narayan](https://github.com/se-edu/addressbook-level4/pulls?q=is%3Apr+author%3Aokkhoy)
-* [Sam Yong](https://github.com/se-edu/addressbook-level4/pulls?q=is%3Apr+author%3Amauris)
\ No newline at end of file
+#### [Lai Yingchen](https://github.com/yingchen0505) <br>
+<img src="images/Yingchen.JPG" width="150"><br>
+Role: Integration, Developer <br>  
+Responsibilities: UI
```

Comment:
You can also assign the roles of Git expert and Eclipse expert. The member will be responsible for helping out other team members with Git and Eclipse issues respectively.